### DISCUSSION TITLE
SSLKEYLOGFILE mechanism to decrypt tcpdump of aws-iot-device-sdk-cpp-v2

### DISCUSSION THREAD

**Discussion by rathsai** [awslabs/aws-crt-cpp#603] [Q&A] [answered]
Hi,

I wanted to analyse a tcpdump of a IoT Connection made through aws-iot-device-sdk-cpp-v2 to AwsIoT and wanted to the mechanism supported by the SDK to dump SSL session Keys so that it can be used with wireshark to decrypt the traffic and look into the actual activity.
Usually, clients dumps these details in a SSLKEYLOGFILE when provided as an envrionment variable while starting the process. 

Any idea whether anything similar is available in aws-iot-device-sdk-cpp-v2 as well.

Thank you

**Comment by bretambrose** [comment, marked as answer]
The CRT does not support any kind of session key logging functionality.
